lib/client-api/src/story_store.ts

```diff
@@ -333,7 +322,7 @@ export default class StoryStore {
         ...accumlatedParameters,
         argTypes: enhancer({
           ...identification,
-          storyFn,
+          storyFn: original,
```

Comment:
@shilman I think the original version of this was looking at the wrong storyFn?

Reply:
No, the enhancer needs the original story function. Maybe we should be passing that as `original` instead of `storyFn`??

I think this "fix" actually introduces a bug. Because the enhancer can't look at whether the original story function accepts args.

Reply:
OK, I misunderstood. You fixed the bug by reverting that part of the PR.

Reply:
Ok so I just fixed a bug in my own code 👍